plugins/inputs/raindrops/raindrops.go

```diff
@@ -154,7 +153,7 @@ func (r *Raindrops) gatherURL(addr *url.URL, acc telegraf.Accumulator) error {
 		}
 		acc.AddFields("raindrops_listen", lis, tags)
 	}
-	return nil
+	return nil //nolint:nilerr // nil returned on purpose
```

Comment:
I don't see why this is an error anyway? Is it due to the errors (`activeErr` and `queuedErr`)? If so, why not make them local in the loop?

Reply:
Nope, because of that:
`plugins/inputs/raindrops/raindrops.go:156:2  nilerr  error is not nil (line 117) but it returns nil`

Reply:
Will it also complain if you move the error definitions to inside the `for` loop?

Reply:
Unfortunately, it will :(

Reply:
I don't see a better way to do this, ignoring the lint sounds fine for now